client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx

```diff
@@ -76,7 +79,25 @@ const InteractionEventForm: React.FC<Props> = (
 
   const onSubmit = (data: InteractionEventDialogData) => {
     const interactionDataToSave = convertData(data);
-    saveInteractions(interactionDataToSave);
+    let allContactsIds: IdToCheck[] = [];
+    let newIds: IdToCheck[] = [];
+    interactions.forEach((interaction: InteractionEventDialogData) => {
```

Comment:
You can map the interactions to array of contacts arrays and then reduce it to single array

Reply:
done